user-guide/live-traffic/inspectors/inspector-types.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Inspector Types
+slug: inspector-types
+tags: Fiddler inspectors types, request, response, HTTP headers, secure requests, headers, textview, webforms, cookies, raw, XML
+publish: true
+position: 20
+---
+
+# Inspector Types
+
+The Inspectors allow you to view the content of each session's request and response in a variety of different formats. Fiddler supports multiple inspector types for the sessions' request (at the top) and response (at the bottom).
+
+## Headers
+
+The __Headers__ inspector, allows you to view the HTTP headers of the request and the response. By default, the content of the __Headers__ inspector is read-only and can't be edited.
+
+![Headers Inspector](../../../images/livetraffic/inspectors/inspectors-headers.png)
+
+There are some conceptual differences between Request Headers and Response Headers, as listed below.
+
+- Request Headers
+
+Every HTTP request begins with plaintext headers that describe what resource or operation is sought by the client.The first line of the request (the _"Request line"_) contains three values: the HTTP method (e.g. __GET__), the URL path which is being requested (e.g. __"/index.html"__), and the HTTP version (e.g. __"HTTP/1.2"__). Request line can be one of one or more lines containing name-value pairs of metadata about the request and the client, such as the _User-Agent_ and _Accept-Language_.
+
+- Response Header
+
+Similar to an HTTP request, every HTTP response begins with plaintext headers that describe the result from the request. The first line of the response (the _"Status line"_) contains the HTTP version (e.g. __"HTTP/1.1"__), the response status code (e.g. __"200"__), and the response status text (e.g. __"OK"__). Following the _Status line_ are one or more lines containing name-value pairs of metadata about the response and the server, such as the length of the response file, the Content-Type, and information about how the response may be cached.
+
+
+## Text
+
+The __Text__ inspector allows you to view the request and response bodies as text. The inspector truncates its display at the first null byte it finds, and as such, is unsuitable for displaying binary content.
+
+Most of the inspector is a large text area that displays the body text interpreted using the character set detected using the headers, the byte-order-marker, or an embedded META tag declaration.
+
+![Text Inspector](../../../images/livetraffic/inspectors/inspectors-textview.png)
+
+
+## Image
+
+The __Image__ inspector (__Response only__) allows you to view the response bodies as an image. The inspector can display most common web image formats, including JPEGs, PNGs, and GIFs, as well as less common formats including cursors, WebP, JPEG-XR, bitmaps, TIFF. The inspector does not support display or SVG graphics (SVG responses can be viewed using the __Web__ inspector).
```

Comment:
1) JPEG, PNG, GIF, without the "s" on each because you have said "formats, including..." and those are the formats.
2) Consider changing "including" in "formats including cursors, WebP" to **like**
3) The inspector does not support display or SVG graphics -> Is that correct or it should be "does not support displaying SVG graphics"?